docs/src/pages/components/masonry/masonry.md

```diff
@@ -0,0 +1,42 @@
+---
+title: React Masonry component
+components: Masonry
+githubLabel: 'component: Masonry'
+---
+
+# Masonry
+
+<p class="description">With horizontal ordering and column stacking, Masonry lays out content of different sizes without leaving gaps.</p>
+
+Masonry maintains a list of content blocks with a consistent width but variable height. The contents are ordered from left to right and top to bottom. If a row is already filled with the specified number of columns, the next item starts another row starts, and it is added to the shortest column.
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+## Basic Masonry
+
+{{"demo": "pages/components/masonry/BasicMasonry.js", "bg": true}}
```

Comment:
Let's try to keep the demo short, so that it can be previewed. Then we wouldn't need the code snippet below